vcs/vcspath_test.go

```diff
@@ -0,0 +1,76 @@
+// Copyright 2015 The Serulian Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package vcs
+
+import (
+	"fmt"
+	"testing"
+)
+
+var _ = fmt.Printf
+
+type pathSuccessTest struct {
+	path           string
+	url            string
+	branchOrCommit string
+	tag            string
+	subpackage     string
+}
+
+var successTests = []pathSuccessTest{
+	pathSuccessTest{"github.com/some/project", "github.com/some/project", "", "", ""},
```

Comment:
Oh, interesting. I think the type does make it a bit more clear, but definitely good to know for smaller types.